src/components/HubPanelWrap.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-col md="4" v-packery-item class="panel-header packery-item mt-2">
+  <b-col md="4" data-packer-item="true" class="panel-header packery-item mt-2" style="padding-top: 10px">
```

Comment:
```suggestion
  <b-col md="4" data-packer-item="true" class="panel-header mt-2">
```

let's move the `padding-top: 10px` to a css style so we can control all that together